### PR TITLE
Added recipes for r-minicran and r-rpostgresql

### PR DESCRIPTION
**Pull request by mariusvniekerk**

### CODE REVIEW

**Comment by mariusvniekerk** [comment]
@conda-forge-admin 
I think these are failing due to the compiler changes?

File: recipes/r-rpostgresql/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+    - r-dbi >=0.3
+    - toolchain
+    - posix                # [win]
+    - gcc                  # [not win]
+    - postgresql
+
+  run:
+    - r-base
+    - r-dbi >=0.3   # [not win]
+    - libpq
```
Comment (isuruf) [inline recipes/r-rpostgresql/meta.yaml]: Add `libgcc` here

**Comment by mariusvniekerk** [comment]
Is that a new standard thing to do.

**Comment by isuruf** [comment]
No, that's always being the standard. If you use `gcc` package in `build`, you have to have `libgcc` in `run`

**Comment by jdblischak** [comment]
Restarted Travis. @bgruening could you please restart CircleCI?

**Comment by bgruening** [comment]
Done!

**Comment by mariusvniekerk** [comment]
@conda-forge-admin seems that the circle ci builds are failing due to a conda-build-all issue?

**Comment by isuruf** [comment]
@mariusvniekerk, can you push an empty commit?

**Comment by bgruening** [comment]
@mariusvniekerk or please remove the comments from your recipes.
Please also consider to use this script for the generation of r-* packages.

https://github.com/bgruening/conda_r_skeleton_helper/blob/master/extra.yaml

It will help you to clean up and adds a few more maintainers.
Thanks!

**Comment by mariusvniekerk** [comment]
Yeah think this is failing due to compiler incompatibilities between r_base (anaconda) and r_base (conda-forge)

**Comment by jdblischak** [comment]
@mariusvniekerk You are correct that the CircleCI build failed because it tried to build with R 3.4.2 from defaults. This should be fixed now that https://github.com/conda-forge/staged-recipes/pull/4407 has been merged.

**Comment by mariusvniekerk** [comment]
so something changed that has now broken windows?

**Comment by jdblischak** [comment]
Looks like the Windows build is now using a lot of packages from the defaults channel:

https://ci.appveyor.com/project/conda-forge/staged-recipes/build/1.0.18698/job/rdmr613w8xaffokv#L169

And it's nothing specific to your recipe, this same problem broke the master branch of staged-recipes earlier today:

https://ci.appveyor.com/project/conda-forge/staged-recipes/build/1.0.18685/job/pobtdkc27iujx5y5#L172

@ocefpaf I saw that you [skipped Windows entirely](https://github.com/conda-forge/staged-recipes/commit/5d92f29337a3daa1e8539f848b6b89e3fb882d0f) on your recently merged R recipes. Do you know what is causing the problem?

**Comment by mariusvniekerk** [comment]
Actually the key difference seems to be that it is now installing R packages from conda-forge and not defaults!

**Comment by jdblischak** [comment]
> https://ci.appveyor.com/project/conda-forge/staged-recipes/build/1.0.18600/job/jax18uwsjdgj3saj#L1464

It's installing R packages from defaults in the build you linked because it was building with R 3.4.2 (only available on defaults). If you look earlier in the build when it is building with R 3.4.1 or R 3.3.2, the R packages are coming from conda-forge.

https://ci.appveyor.com/project/conda-forge/staged-recipes/build/1.0.18600/job/jax18uwsjdgj3saj#L575

Though looking at this build tipped me off to a problem. Your recipe is pulling r-igraph from defaults:

https://ci.appveyor.com/project/conda-forge/staged-recipes/build/1.0.18600/job/0jhnh7vkoqwhe9i3#L586

And that is because r-igraph is skipped on Windows on conda-forge:

https://github.com/conda-forge/r-igraph-feedstock/blob/master/recipe/meta.yaml#L20

Even though that build has passed in some of your previous commits, knowingly relying on dependencies from defaults is dangerous.

https://conda-forge.org/docs/recipe.html#avoid-dependencies-outside-of-conda-forge

Thus I'd recommend skipping Windows entirely as well. Do you need the Windows build?

**Comment by isuruf** [comment]
See my comment here, https://github.com/conda-forge/r-base-feedstock/commit/2b742775c63f552b531a4d60f185a10e1adb4b8d#comments

**Comment by ocefpaf** [comment]
I'm unsure what is the culprit there. I'll be away tomorrow but I'll check
it once I get back.

**Comment by isuruf** [comment]
Pulling `r-base-3.3.2_6` seems to have fixed the windows builds.

**Comment by ocefpaf** [comment]
Thanks @isuruf I'll try to check the pinnings there to see what could be the culprit. Although, for my own use I only need `3.4.1`, so if that one is working I am tempted to leave it alone.

**Comment by mariusvniekerk** [comment]
wheee!, all green

**Comment by mariusvniekerk** [comment]
@ocefpaf seems to be good now

**Pull request merged**